src/common/InjectedComponent.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import * as React from 'react';
+
+class InjectedComponent<
+  TInjectedProps,
+  TProps = {},
+  TState = void,
```

Comment:
idk, for me it should be emty object as default but I found that in core react types, they default it to `void`